CSS/aboutme.css

```diff
@@ -0,0 +1,32 @@
+.about-me {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  margin: 106px 24px;
+  padding: 0;
+}
+
+.title-container {
+  width: 270px;
+  height: auto;
+}
+
+.welcome-txt {
+  font-family: 'Poppins', Arial, Verdana, Tahoma, sans-serif;
+  font-style: normal;
+  font-size: 40px;
+  line-height: 52px;
+  margin: 12px 0;
+}
+
+.description-bg {
+  background-color: #fff;
+}
+
+.decription {
+  font-family: 'Poppins', Arial, Verdana, Tahoma, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 24px;
+}
```

Comment:
- [ ] Kindly update your design to make it match the Figma design, fonts, sizes, and colors currently do not match the design.

## Your design:
![image](https://user-images.githubusercontent.com/50721479/185255736-cb3f8c94-88dd-4fcd-ac27-d84864f0b42d.png)

## Figma design: 
![image](https://user-images.githubusercontent.com/50721479/185255767-d04c1444-5b63-4c29-a784-ed6be69bd5a1.png)